tooling/storybook-addon/src/feature/decorator/ChakraProviderDecorator.tsx

```diff
@@ -23,7 +20,7 @@ export function ChakraProviderDecorator(
   return (
     <ChakraProvider {...chakraParams} theme={themeWithDirectionOverride}>
       <ColorModeSync />
-      <Story {...context} />
+      {story()}
```

Comment:
This may have introduced some breaking changes. The stories here don't render properly.

https://chakra-ui-storybook-4rz4fk7zo-chakra-ui.vercel.app/?path=/story/system-css--css-vars

Reply:
Could you check this and maybe revert that change? @itkrt2y

Reply:
I reverted the story function but not fixed. I will look into it more.